src/crimson/mon/MonClient.cc

```diff
@@ -133,7 +133,8 @@ seastar::future<> Connection::renew_tickets()
 {
   if (auth->need_tickets()) {
     return do_auth(request_t::general).then([](auth_result_t r) {
-      if (r != auth_result_t::success)  {
+      if (r != auth_result_t::success && r != auth_result_t::canceled)  {
```

Comment:
```suggestion
      if (r == auth_result_t::failure)  {
```

Reply:
i think `failure` is designed for this use case, i.e., to denote the "failure".

Reply:
Switched!